templates/hub.html

```diff
@@ -24,6 +24,7 @@ <h1 class="u-mt-m">{{ content.title }}</h1>
         {
           "groups": [
             {
+              "headers":["Name of section or person", "Section Progress", "Access Section"],
```

Comment:
The capitalisation doesn't match the card:
```suggestion
              "headers":["Name of section or person", "Section progress", "Access section"],
```